### ISSUE TITLE
Disappear_time is from july 19th

### ISSUE THREAD

**Issue by shiznewski** [AHAAAAAAA/PokemonGo-Map#3077]
here is the the data my webhook gets from time to time. the disappear time is from july 19th when i got to an epoch convert and put in the time. 

{"message": {"time_until_hidden_ms": -1215716200, "last_modified_time": 1470094531431, "disappear_time": 1468893215.0, "pokemon_id": 19, "latitude": 39.752150188893296, "spawnpoint_id": "89c6d1327a9", "encounter_id": "MTM0NjA0ODQ2Nzg0ODM5MzU0ODU=", "longitude": -75.09556182092717}, "type": "pokemon"}

**Comment by plasticlobster** [comment]
According to someone on Reddit who investigated this in-depth, a negative disappear time means it's a 60-minute spawn.

https://www.reddit.com/r/pokemongodev/comments/4uuwx0/spawn_myths_debunked_aka_the_truth_about_spawn/

**Comment by shiznewski** [comment]
Oh. I figured 1. It was a bug. 2. This was ninantics way of catching bots. If one of the boys tried to encounter that encounter id it would be an easy way to spot and ban accounts that were leveling up by a bot.

**Comment by andrewda** [comment]
I'll mark this as a bug so we can properly address the issue. We should notify the user that this is most likely a 45 or 60 minute spawn.

**Comment by Alex--C** [comment]
These spawns definitely do show up in the official PoGo App. So it should not be any type of bot detection as mentioned by @shiznewski . I'm not sure about the spawn duration though, I did not yet try that out. Maybe I'll have time to try it with the next spawn of this kind.

**Comment by joelverhagen** [comment]
Yes, I observed this issue in my response on https://github.com/AHAAAAAAA/PokemonGo-Map/issues/2703. Could this be related to the fact that the `encounter_id` is not unique?

**Comment by Alex--C** [comment]
Seems like in my case, I received a "normal" encounter a few minutes after the one with the July 19th timestamp. Same encounter-id. This second encounter then did show up on the map correctly, with a 15min despawn timer. I think I'm just going to ignore that specific disappear time in my webhooks handler until we completely understand what is going on here.
